src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java

```diff
@@ -46,45 +51,27 @@
  *
  * <h4>Step 1. Run Mutect2 in tumor-only mode for each normal sample.</h4>
  * <pre>
- * gatk Mutect2 \
- *   -R reference.fa \
- *   -I normal1.bam \
- *   -tumor normal1_sample_name \
- *   -O normal1_for_pon.vcf.gz
+ * gatk Mutect2 -R reference.fasta -I normal1.bam -O normal1.vcf.gz
  * </pre>
  *
- * <h4>Step 2. Create a file ending with .args or .list extension with the paths to the VCFs from step 1, one per line.</h4>
- * <p>This approach is optional. Other extensions will error the run. </p>
+ * <h4>Step 2. Create a GenomicsDB from the normal Mutect2 calls.</h4>
+ * Note that GenomicsDBImport is currently (as of February 2019) inefficient when processing multiple intervals.  Therefore,
```

Comment:
Depends on what your data looks like.  For exomes it should be fine now: if you use `--merge-input-intervals` https://github.com/broadinstitute/gatk/pull/5540

Reply:
@ldgauthier The M2 pon wdl I wrote for this PR scatters over contigs, making a chr1 pon from a chr1 GenomicsDB, a chr2 pon from a chr2 GenomicsDB, and then merging.  Are you saying that instead I should make a single GenomicsDB with `--merge-input-intervals` and create a single pon from that DB?  Is this true even if it's a WGS pon and there are lots of variants?

Reply:
~24 contigs is probably not a big deal. (That arg also won't help WGS because each contig has to get its own interval anyway.)  Exomes were the real pathological case because there's a very significant startup cost for each interval and even for scattered exomes we had O(1000) intervals.